scriptworker/constants.py

```diff
@@ -149,13 +149,19 @@
 
     # decision task cot
     "valid_decision_worker_types": (
-        "gecko-decision",
+        "gecko-1-decision",
+        "gecko-2-decision",
+        "gecko-3-decision",
+        "gecko-decision", # legacy
     ),
 
     # docker-image cot
     "valid_docker_image_worker_types": (
         "taskcluster-images",   # TODO: Remove this image once docker-images is the only valid worker type
-        "gecko-images",
+        "gecko-images", # legacy
```

Comment:
Same here.
Also, for the above `TODO`, do you know if there's a timeframe or bug to follow for when `taskcluster-images` can be removed?